internal/configtxgen/encoder/encoder.go

```diff
@@ -672,3 +678,35 @@ func (bs *Bootstrapper) GenesisBlock() *cb.Block {
 func (bs *Bootstrapper) GenesisBlockForChannel(channelID string) *cb.Block {
 	return genesis.NewFactoryImpl(bs.channelGroup).Block(channelID)
 }
+
+func addBFTBlockPolicy(consenterProtos []*cb.Consenter, ordererGroup *cb.ConfigGroup) {
```

Comment:
can we name the first parameter `consenterProtos` as `consenters` for code readability.